dev/io.openliberty.microprofile.telemetry.logging.internal.common/src/io/openliberty/microprofile/telemetry20/logging/internal/OpenTelemetryLogHandler.java

```diff
@@ -180,7 +186,18 @@ private void formatEvent(LogRecordBuilder builder, String source, Object event)
     public void synchronousWrite(Object event) {
         OpenTelemetryInfo otelInstance = null;
 
-        if (OpenTelemetryAccessor.isRuntimeEnabled()) {
+        if (!CheckpointPhase.getPhase().restored()) {
+            // restored() true if checkpoint has been restored or was never active.
+            // so with the negation we enter this block if checkpoint is active and
+            // we have not yet restored.
+
+            // In that situation OpenTelemetry always returns a no-op OpenTelemetryInfo
+            // but we don't want to start poking around the internals before they're set
+            // up in the post-restore phase.
+
+            // in short, this block is no-op.
+
```

Comment:
Shouldn't be needed due to `@SatisfyingConditionTarget("(" + Condition.CONDITION_ID + "=" + CheckpointPhase.CONDITION_PROCESS_RUNNING_ID + ")")` on the class.

If I'm understanding correctly, that should prevent the component from starting when you're going to take a checkpoint so you should never hit this condition.